databuilder/models/table_elasticsearch_document.py

```diff
@@ -19,7 +19,8 @@ def __init__(self,
                  column_descriptions,  # type: List[str]
                  total_usage,  # type: int
                  unique_usage,  # type: int
-                 tags,  # type: List[str]
+                 tags,  # type: List[str],
+                 badges,  # type: List[str]
```

Comment:
let's make badges=None to make it backward compatible. Otherwise I think the sample dataloader will fail.

Reply:
good catch!